### PR TITLE
Additional if statement to solve the problem of 2 units

### LINKED ISSUE #3: Binary classification with 2 units gives wrong precision and recall
Hi, I've seen the similar issue that was about evaluation after training on multi-class classification, mine is about the training. If one has only one unit at the end, everything is correct, however if dense layer with two units is used for the last layer with a softmax activation, true positive is always equals to true negative, and false positive is always equals to false negative in every batch during the training. And when you sum all of them you obtain a number that is exactly equals to twice of batch size. Please kindly check my code below.
Note: I'm aware of the fact that these metrics shouldn't be checked per batch basis, but seeing something wrong pushed me to create this issue. By the way, thanks for the library and effort.

https://colab.research.google.com/drive/1lmQ-hWcN4tsGMicd4dKnSjeTD-BdgJuE

### PR DESCRIPTION
**Pull request by Avcu**
Since to_categorical function always put the true labels into second column when used with 2 classes, an additional if statement can be added in case of usage of more than one unit at the last layer.

Closes #3

### CODE REVIEW

**Comment by Avcu** [comment]
For the example code, kindly refer the link below
https://colab.research.google.com/drive/1jeRnEJlqT82OT9Tibviq3JTo_8qOPnql#scrollTo=rErzh9evOiSY

**Comment by ybubnov** [comment]
@Avcu , thank you for submitting this PR!

File: keras_metrics/metrics.py [hunk 1]
```diff
@@ -6,6 +6,11 @@
 def _int32(y_true, y_pred):
     y_true = K.cast(y_true, "int32")
     y_pred = K.cast(K.round(y_pred), "int32")
+    
+    if y_pred.shape[1] != 1:
+      y_pred = y_pred[:,1:2]
+      y_true = y_true[:,1:2]
```
Comment (ybubnov) [inline keras_metrics/metrics.py]: From what I understand, this code forces us to calculate the metrics only for a _second_ label.

Comment (ybubnov) [reply keras_metrics/metrics.py]: What if instead of making it explicitly during the conversion step, this parameter will be passed to the constructor of the metric itself, like:
```py
recall = keras_metrics.recall(label=2)
```

Comment (Avcu) [reply keras_metrics/metrics.py]: discussed here https://github.com/netrack/keras-metrics/pull/5
